src/coreComponents/common/fixedSizeDeque.hpp

```diff
@@ -0,0 +1,158 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+#ifndef FIXEDSIZEDEQUE_HPP
+#define FIXEDSIZEDEQUE_HPP
+
+#if defined( GEOSX_USE_CUDA )
```

Comment:
I think a lot of these includes are no longer needed.

Reply:
I'll check.